lib/rainforest/cli/options.rb

```diff
@@ -67,10 +67,15 @@ def initialize(args)
             @custom_url = value
           end
 
+          opts.on("--run-description DESCRIPTION", "Add a description for the run been started.") do |value|
```

Comment:
`Add a description for the run been started` -> `Add a description for the run.`